pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -359,6 +359,36 @@ def create_model(df_sets, df_parameters, default={}):
     model.s_B = Set(
         initialize=model.df_sets["TreatmentTechnologies"], doc="Treatment Technologies"
     )
+    model.df_parameters["LLA"] = {
+        **model.df_parameters["PNA"],
+        **model.df_parameters["CNA"],
+        **model.df_parameters["CCA"],
+        **model.df_parameters["NNA"],
+        **model.df_parameters["NCA"],
+        **model.df_parameters["NKA"],
+        **model.df_parameters["NSA"],
+        **model.df_parameters["NRA"],
+        **model.df_parameters["FCA"],
+        **model.df_parameters["RCA"],
+        **model.df_parameters["RNA"],
+        **model.df_parameters["SNA"],
+        **model.df_parameters["RSA"],
+        **model.df_parameters["SCA"],
+    }
```

Comment:
This list seems incomplete to me - is there a reason that the following types of arcs are not included:

- PCA
- PPA
- NOA
- RKA
- SKA
- SRA
- SOA

Reply:
These tabs do not exist yet in our input sheet, so df_parameters does not have a key for these arcs. If/when these arcs are added to the input, they should be added to this list.

Reply:
I see - that makes sense. Thanks for explaining. In that case, you can discard my comment below about needing to add these arcs to the construction of the LLA set.